misk/src/main/kotlin/misk/jdbc/Cluster.kt

```diff
@@ -0,0 +1,17 @@
+package misk.jdbc
+
+/**
+ * [Cluster] provides a consistent abstraction for
+ * interacting with various database-related resources [T]
+ * based on their role in the underlying database topology.
+ *
+ * Database clusters are often composed of a Read/Write master
+ * instance and a set of Read-Only replica instances.
```

Comment:
I would make it clear that even if the actual cluster has a set of replicas, Misk leaves it up to the underlying persistence layer to provide a single addressable endpoint that manages connections to the set